src/Components/MedicalHistory/ModalDelete.jsx

```diff
@@ -0,0 +1,58 @@
+import { useContext } from "react";
+import { Modal, Button } from "react-bootstrap";
+import { GetTheAppContext } from "../../Context/AppContext";
+import { statusDeleted } from "../../Context/HTTPStatus";
+
+export const ModalDelete = ({ show, handleClose }) => {
+  const {
+    setTextAlert,
+    handleShowFloatAlter,
+    deleteHistoryFunction,
+    dataMedicalHistory,
+    setAllMedicalHistoryData,
+    allHistoryFromApiFunction,
+    setActionButtonModal,
+  } = useContext(GetTheAppContext);
+
+  const buttonClick = () => {
+    handleClose();
+  };
+
+  return (
+    <Modal show={show} onHide={handleClose}>
+      <Modal.Header closeButton>
+        <Modal.Title>Confirmar eliminación</Modal.Title>
+      </Modal.Header>
+      <Modal.Body>
+        <p>¿Está seguro de que desea eliminar este historial?</p>
+      </Modal.Body>
+      <Modal.Footer>
+        <Button variant="secondary" onClick={handleClose}>
+          Cancelar
+        </Button>
+        <Button
+          variant="danger"
+          onClick={async () => {
+            setActionButtonModal("Eliminar");
+            const response = await deleteHistoryFunction(dataMedicalHistory.id);
+
+            if (response.status === statusDeleted) {
+              await allHistoryFromApiFunction(setAllMedicalHistoryData);
+              buttonClick();
+              setTextAlert(
+                `Se eliminó el historial médico del paciente ${dataMedicalHistory.patient.name}`
+              );
+              handleShowFloatAlter();
```

Comment:
```suggestion
              showFloatAlter();
```

si el nombre de la función cambio entonces tambien debe cambiar el nombre donde sea llamado, buscar los lugares en este pr donde aparece handle en el nombre de funciones y removerlos, recuerden usar la recomendación de iniciar los metodos con un verbo segun lo que realicen

Reply:
Como es un función genérica handleShowFloatAlter se ocupa en todos los componentes si hago el cambio la aplicación dejara de funcionar , PD aquí aun no se contemplan los cambios indicados ya que este PR se mando a revisión antes de que se iniciara con los cambios que nos indico en la hoja